app/src/main/res/values/strings.xml

```diff
@@ -653,4 +654,7 @@
     <string name="Search_Bar_Hint_Text">Search..</string>
     <string name="search_menu_title">Search</string>
 
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```

Comment:
What is this one for?

Reply:
I actually didn't coded this manually it was automatically added! Shall I try removing it?